backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -711,6 +711,15 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
         "sequence(id+10, id, -3) from range(1)")(checkOperatorMatch[ProjectExecTransformer])
   }
 
+  test("Bug-398 collec_list failure") {
```

Comment:
please check why it will be failed when add this test case in `GlutenClickHouseTPCHSuite` for the mergetree

Reply:
vanilla spark doesn't support mergetree, we cannot use `compareResultsAgainstVanillaSpark` to check the result.